### DISCUSSION TITLE
Provide KONG support for RISCV

### DISCUSSION THREAD

**Discussion by Rakshithakrishna2** [Kong/kong#7984] [Ideas and feature requests]
Hi,

I am facing while porting kong to RISCV as Kong depends on luaJIT and openresty, which do not work on riscv (but do work on riscv32).
Kindly provide the support for RISCV.

**Comment by kikito** [comment]
Hello, thank you for your feedback. We unfortunately don't maintain either LuaJIT or Openresty, we can't comply with your request at this time.
